config/initializers/extends.rb

```diff
@@ -409,6 +409,12 @@ class Extends
     shared_write: 2
   }.freeze
 
+  SHARED_OBJECTS_PERMISSION_LEVELS = {
+    not_shared: 0,
```

Comment:
Why is the not_shared option needed? Can we have shared objects that are not shared?

Reply:
it was originally used in repository model

Reply:
removed